app/main.py

```diff
@@ -1,9 +1,17 @@
-from fastapi import FastAPI
+from fastapi import APIRouter, FastAPI
 
-from app.v1.endpoints import v1_router
+from app.v1.released.endpoints import v1_router as released_v1_router
+from app.v1.upcoming.endpoints import v1_router as upcoming_v1_router
 
 # Initialize FastAPI app
 app = FastAPI()
 
-# Include the routers under versioned paths
-app.include_router(v1_router, prefix="/v1")
+# Create a main API router with the /api prefix
+api_router = APIRouter()
+
+# Include individual service routers under the main API router
+api_router.include_router(released_v1_router, prefix="/v1/released", tags=["released"])
```

Comment:
yeah it makes sense! I didn't really stop to think about the names much, well I named it based on frontend tabs :sweat_smile:  I will use

- `v1/release-notes`
- `v1/upcoming-changes` - because it might not be just EOL info